src/html5sortable.ts

```diff
@@ -552,6 +556,13 @@ export default function sortable (sortableElements, options: configuration|objec
         store(sortableElement).placeholder.style.height = draggingHeight + 'px'
         store(sortableElement).placeholder.style.width = draggingWidth + 'px'
       }
+      // Keep some CSS class fo the dragged element, to keep style.
+      // Usefull if sortable elements have different dimensions according class.
+      options.keepCssClass.forEach(className => {
```

Comment:
I think from a performance point of view you should first get the elements you want to add. Maybe use filter to get the elements that are in both arrays.

Afterwards just like with remove, you can `add` all at once. DOM interactions are often expensive.
`add(...comparedArray)`

Reply:
Sorry you lost me in the first sentence.
Second sentence nice did't know that too.